templates/help.html

```diff
@@ -142,6 +141,21 @@ <h2 id="naming-scheme">Naming Scheme</h2>
   <p>
     Subtitle filenames should generally have the same name as the filename of the video it came from, except with an <code>.srt</code> or <code>.ass</code> extension instead of the video extension. This allows users to quickly identify what video the subtitles originated from, and hopefully allows them to consistently modify the timing relative to their own files.
   </p>
+  <p>
+    For rips directly from streaming sources or Blu-ray, <strong>include the name of the source</strong> and consider standardizing the filename to match the other files on the site for the convenience of the user. If the release uses a different numbering scheme than the standard one as sometimes the case with Netflix, also consider including the title of the episode. Examples of common formats:
+    <ul>
+      <li><code>Kaguya-sama Love Is War.S02E01.Netflix.CC.ja.srt</code></li>
+      <li><code>Pokemon Horizon.S01E01.Amazon.ja[sdh].srt</code></li>
+      <li><code>史上最強の大魔王、村人Aに転生する.S01E01.村人A.WEBRip.Netflix.ja[cc].srt</code></li>
+      <li><code>モンティ・パイソン・アンド・ホーリー・グレイル.BDrip.ja.srt</code></li>
+    </ul>
+  </p>
+  <p>
+    At the bare minimum, retimed files should be prefixed with <code>[Group]</code> so users can distinguish what release they are meant to be used for. Files can also be suffixed with <code>(retimed from Source)</code> to help users discern if the subtitles are accurate (Kamigami sometimes is not), if there are closed captions, and what the ASS styling might look like if applicable. (Example: <code>[Vodes] Jujutsu Kaisen 0 (2021) [v2] [BD 1080p HEVC Dual-Audio] (retimed from Netflix).srt)</code>
```

Comment:
```suggestion
    At the bare minimum, retimed files should be prefixed with <code>[Group]</code> so users can distinguish what release they are meant to be used for. Files can also be suffixed with <code>(retimed from Source)</code> to help users discern if the subtitles are accurate, if there are closed captions, and what the ASS styling might look like if applicable. (Example: <code>[Vodes] Jujutsu Kaisen 0 (2021) [v2] [BD 1080p HEVC Dual-Audio] (retimed from Netflix).srt)</code>
```

I'd rather not explicitly name anything as being "bad". That being said, I'm not sure what this is saying after "accurate".

Reply:
A better phrasing is
"Files can also be suffixed with <code>(retimed from Source)</code> to help users discern where the subtitles come from if they have issues. In addition, users can reference the name of the source file to ascertain whether the file contains closed captions and what ASS styling it might have.

Reply:
Ah yeah that reads better.

Reply:
```suggestion
    At the bare minimum, retimed files should be prefixed with <code>[Group]</code> so users can distinguish what release they are meant to be used for. Files can also be suffixed with <code>(retimed from Source)</code> to help users discern where the subtitles come from if they have issues. In addition, users can reference the name of the source file to ascertain whether the file contains closed captions and what ASS styling it might have. (Example: <code>[Vodes] Jujutsu Kaisen 0 (2021) [v2] [BD 1080p HEVC Dual-Audio] (retimed from Netflix).srt</code>)
```